service/src/main/java/org/apache/celeborn/server/common/service/config/DynamicConfigServiceFactory.java

```diff
@@ -47,4 +49,9 @@ public static ConfigService getConfigService(CelebornConf celebornConf) {
 
     return _INSTANCE;
   }
+
+  @VisibleForTesting
+  public static void reset() {
+    _INSTANCE = null;
```

Comment:
Need call shutdown method before reset the _INSTANCE

Reply:
@RexXiong updated the PR